flink-dist/src/main/flink-bin/bin/pyflink2.bat

```diff
@@ -22,4 +22,4 @@ setlocal EnableDelayedExpansion
 SET bin=%~dp0
 SET FLINK_ROOT_DIR=%bin%..
 
-%FLINK_ROOT_DIR%\bin\flink run -v %FLINK_ROOT_DIR%\lib\flink-python*.jar 2 %*
\ No newline at end of file
+"%FLINK_ROOT_DIR%\bin\flink" run -v "%FLINK_ROOT_DIR%"\lib\flink-python*.jar 2 %*
```

Comment:
I was thinking of quotes around `%*`. This is necessary in the shell scripts.

If you haven't already, could you try something like WordCount with spaces in the output file name?